src/xcode/ENA/ENA/Source/Workers/Logging/Logging.swift

```diff
@@ -37,6 +37,9 @@ extension OSLog {
 	static let traceLocation = OSLog(subsystem: subsystem, category: "traceLocation")
 	/// QR Code
 	static let qrCode = OSLog(subsystem: subsystem, category: "qrCode")
+	/// Vaccination
+	static let vaccination = OSLog(subsystem: subsystem, category: "vaccination")
```

Comment:
```suggestion
	static let 💉 = OSLog(subsystem: subsystem, category: "vaccination")
```
😅

Reply:
how do we type this in code so i can change it in other places too :D ?